ios/RNSScreenStackHeaderConfig.mm

```diff
@@ -756,9 +756,22 @@ - (void)mountChildComponentView:(UIView<RCTComponentViewProtocol> *)childCompone
 - (void)unmountChildComponentView:(UIView<RCTComponentViewProtocol> *)childComponentView index:(NSInteger)index
 {
   [_reactSubviews removeObject:(RNSScreenStackHeaderSubview *)childComponentView];
+  [self setViewToSnapshot:(RNSScreenStackHeaderSubview *)childComponentView];
```

Comment:
I don't know if you should set the view to snapshot after removing childComponentView from `_reactSubviews`. Because of the side-effects, have you tried setting view to snapshot before that?

Reply:
It looks like setting the view to snapshot before works fine. I have not observed any side-efffects in the current solution but I'll trust you on this 😄

Reply:
https://github.com/software-mansion/react-native-screens/pull/2247/commits/a2882895f5a5bd6b3ebba3c8b50aa55d6280cc11